Release 251015

jkroll-deepgram opened the pull request:
## Proposed changes
<!--
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
-->

## Types of changes

What types of changes does your code introduce to the Deepgram self-hosted resources?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update or tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I have tested my changes in my local self-hosted environment
  - Please describe your testing setup and methodology here
- [ ] I have added necessary documentation (if appropriate)

## Further comments

<!--
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
-->

therealevanhenry reviewed (approved):
LGTM

therealevanhenry commented:
@jkroll-deepgram the helm validation step failed with this error:
```
Run if ! git diff --quiet -- ./charts/deepgram-self-hosted/README.md; then
README.md is out of date. Please run helm-docs and commit the changes.
Error: Process completed with exit code 1.
```

jkroll-deepgram commented:
@therealevanhenry thanks! I fixed a different thing and forgot to redo that command. Fixed the helm docs: https://github.com/deepgram/self-hosted-resources/pull/101/commits/989ecf4ef6459b08f67805481c1b14a61463bb37
